tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -160,7 +160,7 @@ async def test_nft_wallet_creation_automatically(
     )
     assert len(txs) == 1
     assert txs[0].spend_bundle is not None
-    await wallet_node_0.wallet_state_manager.add_pending_transaction(txs[0])
```

Comment:
this seems to change behavior. adding only the first transaction vs. all of them